### DISCUSSION TITLE
backend-development-with-directus/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#59] [Blog comments]
# backend-development-with-directus/

https://blogs.taiga.nl/martijn/posts/backend-development-with-directus/

**Comment by disqus-comments-migrator** [comment]
<em>[Simon Boddy](https://disqus.com/by/simonboddy/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at mei 31 2022, 03:35</em>

---

<p>Looks really good ! Thanks.</p>

**Comment by Torwald45** [comment]
It's been a while since the post was published - what do you think about Directus in retrospect... it's also worth noting that it's no longer open source.

**Reply by martijnboland** [reply]
In retrospect, I think Directus has actually improved a lot in the last years. New releases are stable and a lot of new functionality is added (such as content versioning and live preview). What hasn't changed is that our customers still like working with the Directus UI.

We have used Directus for new projects in the last few years and continue to do so. Over time we figured out a nice way to setup our development and production environments to enable CI/CD and easy extension development.

Regarding the licensing: it's Open Source and everyone can still contribute, however, the license was changed from GPL v3 to BSL 1.1. Effectively this means that for a company with a yearly turnover less than 5 million USD, Directus remains free to use. This is done to create a more sustainable business model for Directus while staying as close as possible to the Open Source values.
